CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   Guidelines but will shortly.  
   [JP Simard](https://github.com/jpsim),
   [Norio Nomura](https://github.com/norio-nomura)
+  
+* `variable_name` rule is now `identifier_rule` as it validates other
```

Comment:
`identifier_name`. Also, it'd be nice to keep `VariableNameRule` around, logging a warning to the console if used in comment commands or the config file, proxying its implementation to `IdentifierNameRule`. Would lead to a nicer deprecation path.